helm-chart/templates/app.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: workload.crossplane.io/v1alpha1
+kind: KubernetesApplication
+metadata:
+  name: {{ .Release.Name }}-app
```

Comment:
Previously, we were able to create multiple wordpresses in the same namespace by using different names in the wordpressinstance CR. Can we still do that if we're using `.Release.Name` for uniqueness?

Reply:
Yes, `.Release.Name` is simply the name of the wordpressinstance CR. The only new thing is that the `Namespace` resource in remote cluster is named with `.Release.Name`, too. This is in order to ensure that each WordpressInstance that uses the same cluster deploys a separate namespace so that you can reuse the same cluster with different `WordpressInstance` CRs using `UseExistingTarget`.